requirements.txt

```diff
@@ -1,21 +1,3 @@
-# This file is managed centrally by release-tools and should not be modified
```

Comment:
If this is a reactive charm, you have to keep that disclaimer and follow it, i.e. you're not allowed to make special modifications to this file. Any change you need needs to go to https://github.com/openstack-charmers/release-tools/blob/master/global/source-zaza/requirements.txt first. We want to avoid charms from becoming snowflakes, and want to keep the boilerplate of our 60+ OpenStack charms the same. This file can only be a verbatim copy of what is in `release-tools`.